server/src/main/java/org/opensearch/throttling/admissioncontrol/AdmissionControlSettings.java

```diff
@@ -0,0 +1,122 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.throttling.admissioncontrol;
+
+import org.opensearch.common.settings.ClusterSettings;
+import org.opensearch.common.settings.Setting;
+import org.opensearch.common.settings.Settings;
+import org.opensearch.throttling.admissioncontrol.enums.AdmissionControlMode;
+
+/**
+ * Settings related to admission control.
+ * @opensearch.internal
+ */
+public final class AdmissionControlSettings {
+
+    /**
+     * Default parameters for the AdmissionControlSettings
+     */
+    public static class Defaults {
+        public static final String MODE = "disabled";
+    }
+
+    public static final String IO_BASED_ADMISSION_CONTROLLER = "global_io_usage";
+
+    /**
+     * Feature level setting to operate in shadow-mode or in enforced-mode. If enforced field is set
+     * rejection will be performed, otherwise only rejection metrics will be populated.
+     */
+    public static final Setting<AdmissionControlMode> ADMISSION_CONTROL_TRANSPORT_LAYER_MODE = new Setting<>(
+        "admission_control.transport.mode",
+        Defaults.MODE,
+        AdmissionControlMode::fromName,
+        Setting.Property.Dynamic,
+        Setting.Property.NodeScope
+    );
+
+    /**
+     * Feature level setting to force enable feature setting for individual controllers.
+     * Default each admissionController Settings will be considered but once this flag enabled feature level settings
+     * will be considered. Will help during turn off entire feature when its required
+     */
+    public static final Setting<Boolean> ADMISSION_CONTROL_FORCE_ENABLE_DEFAULT_SETTING = Setting.boolSetting(
+        "admission_control.force_enable_defaults",
+        false,
+        Setting.Property.Dynamic,
+        Setting.Property.NodeScope
+    );
+
+    private volatile Boolean forceEnableDefault;
+
+    private volatile AdmissionControlMode transportLayeradmissionControlMode;
+
+    private final AdmissionControlService admissionControlService;
```

Comment:
Can we remove this ? I don't see any usage of this.

Reply:
Removed